screens/Notifications.js

```diff
@@ -1,19 +1,35 @@
 import React from 'react';
-import { StyleSheet, Text, View, Image, Dimensions } from 'react-native';
+import { StyleSheet, Text, View, Image, Dimensions, FlatList } from 'react-native';
 
 
 export default class Notifications extends React.Component {
   render() {
     return (
       <View style={styles.container}>
-        <View style={styles.orangeRectangle}></View>
+        <View style={styles.peachBox}></View>
```

Comment:
`View` components that don't have content inside them, can just be closed like `<View />`